engine/hatchery/marathon/marathon.go

```diff
@@ -39,6 +39,14 @@ func (h *HatcheryMarathon) ApplyConfiguration(cfg interface{}) error {
 		return fmt.Errorf("Invalid configuration")
 	}
 
+	// h.Client = cdsclient.NewService(h.Config.API.HTTP.URL, 60*time.Second)
```

Comment:
Comment are normal, I avoid update to many hatchery for now, waiting branch from @bnjjj 
They will be useful to have hatchery as a CDS Services